src/components/card/FlipCard.spec.js

```diff
@@ -0,0 +1,107 @@
+import { render, screen } from '@testing-library/react'
+import FlipCard from './FlipCard'
+
+describe('FlipCard', () => {
+  const name = 'Natur- und Tierpark Brüggen'
+  const description =
+    'Der Natur- und Tierpark Brüggen ist ein Zoo mit umfangreichem Spiel- und Freizeitbereich!'
+  const street = 'Brachter Str. 98'
+  const city = 'Brüggen'
+  const zipCode = 41379
+  const country = 'Germany'
+  const openingHours = '09:00 - 18:00'
+  const website = 'https://natur-und-tierpark-brueggen.de'
+  const freeOfCharge = false
+
+  it('renders', () => {
+    render(
+      <FlipCard
+        name={name}
+        description={description}
+        street={street}
+        city={city}
+        zipCode={zipCode}
+        country={country}
+        openingHours={openingHours}
+        website={website}
+        freeOfCharge={freeOfCharge}
+      />
+    )
+
+    const title = screen.getByRole('heading')
+
+    expect(title).toBeInTheDocument()
+
+    const link = screen.getByRole('link')
+
+    expect(link).toBeInTheDocument()
+  })
+
+  it('displays the text', () => {
+    render(
+      <FlipCard
+        name={name}
+        description={description}
+        street={street}
+        city={city}
+        zipCode={zipCode}
+        country={country}
+        openingHours={openingHours}
+        website={website}
+        freeOfCharge={freeOfCharge}
+      />
+    )
+
+    const titleAndNameContent = screen.getAllByText(
+      'Natur- und Tierpark Brüggen'
+    )
+
+    expect(titleAndNameContent).toHaveLength(2)
```

Comment:
For better readability I would group each query and `expect`:
```suggestion
    const titleAndNameContent = screen.getAllByText(
      'Natur- und Tierpark Brüggen'
    )
    expect(titleAndNameContent).toHaveLength(2)
```